src/main/java/org/opentripplanner/util/TranslationHelper.java

```diff
@@ -79,22 +81,28 @@ public void importTranslations(
     }
   }
 
+  @Nullable
   public I18NString getTranslation(
-    Class<?> clazz,
-    String fieldName,
-    String recordId,
-    String defaultValue
+    @Nonnull Class<?> clazz,
+    @Nonnull String fieldName,
+    @Nonnull String recordId,
+    @Nullable String defaultValue
   ) {
     return getTranslation(clazz, fieldName, recordId, null, defaultValue);
   }
 
+  @Nullable
   public I18NString getTranslation(
-    Class<?> clazz,
-    String fieldName,
-    String recordId,
-    String recordSubId,
-    String defaultValue
+    @Nonnull Class<?> clazz,
+    @Nonnull String fieldName,
+    @Nonnull String recordId,
+    @Nullable String recordSubId,
+    @Nullable String defaultValue
   ) {
+    if (defaultValue == null) {
+      return null;
+    }
```

Comment:
It seems a bit strange that you are not allowed to specify a translation if the default is `null`, but I guess that is the original behavior.